app/model/lib/r_script.py

```diff
@@ -6,15 +6,16 @@
 import logging
 from pathlib import Path
 
-LOGGER = logging.getLogger()
+_LOGGER = logging.getLogger()
 
 
 class RScript:
     """
-    This object provides a generic interface to executing an R script using an
-    `Rscript` executable found in the PATH. It expects to be given a root
-    directory (likely a temporary one) where it'll look for its input files and
-    produce its outputs.
+    A generic interface to executing an R script.
+
+    It uses the ``Rscript`` executable found in the PATH. It expects to be
+    given a root directory (likely a temporary one) where it'll look for its
+    input files and produce its outputs.
```

Comment:
The general format of documentation strings is: The first line gets shown as a short summary on top, the entire contents get shown below the function.

The first line needs to be standalone or the summary might get cut off in a weird place.